besspin/cyberPhys/canlib/lib/canspecs.h

```diff
@@ -1,13 +1,12 @@
 /*
-* Cyberphys CAN Frames Specification
-* Project: SSITH CyberPhysical Demonstrator
-* Name: ../cyberphyslib/cyberphyslib/canlib/canspecs.py
-* Author: Steven Osborn <steven@lolsborn.com>, Kristofer Dobelstein, Ethan Lew <elew@galois.com>
-* Michal Podhradsky <mpodhradsky@galois.com>
-* Date: 23 April 2021
-* This file was created by BESSPIN-Tool-Suite/besspin/cyberPhys/canlib/make_can_spec.py
-* Version hash: 956273597e1c80b771cde09098766c3a
-*/
+    * Cyberphys CAN Frames Specification
```

Comment:
Is it me, or is this new indentation a bit off?

Reply:
You are right, looks like there is an equivalent of an extra tab.

Reply:
Fixed